pwndbg/heap/structs.py

```diff
@@ -290,6 +291,66 @@ class c_malloc_state_2_26(Structure):
     ]
 
 
+class c_malloc_state_2_12(Structure):
+    """
+    This class represents malloc_state struct for GLIBC < 2.22 as a ctypes struct.
+
+    https://github.com/bminor/glibc/blob/glibc-2.23/malloc/malloc.c#L1686-L1724
```

Comment:
Is this whole docstring valid? It says GLIBC<2.22 but did you really check GLIBCs 2.12 up to 2.23 that we do support?

Also the link points to glibc-2.23 branch, so it seems it maybe is a copy-paste leftover from c_malloc_par_2_23 struct.

Please fix the docstring

Reply:
ye, But the link needs a fix :)